docs/Gemfile.lock

```diff
@@ -139,6 +141,7 @@ DEPENDENCIES
   middleman-deploy (~> 2.0.0.pre.alpha)
   middleman-livereload
   middleman-minify-html
+  middleman-protect-emails
```

Comment:
I stopped trying to be smarter than spammers and show clear emails everywhere. Did not see any raise in spam (gmail spam handles it). Also "support@algolia.com" is already public, right here in github commit and in google https://www.google.fr/search?q=%22support%40algolia.com%22&oq=%22support%40algolia.com%22&aqs=chrome..69i57j69i59.3046j1j7&sourceid=chrome&ie=UTF-8

:D